lib/filterx/filterx-grammar.ym

```diff
@@ -346,6 +346,9 @@ expr_generator_unchecked
 	| list_generator
 	| regexp_search
 	| '(' expr_generator ')'		{ $$ = $2; }
+	| expr '+' expr_generator		{ $$ = filterx_operator_plus_generator_new($1, $3); }
+	| expr_generator '+' expr 		{ $$ = filterx_operator_plus_generator_new($1, $3); }
+	| expr_generator '+' expr_generator 	{ $$ = filterx_operator_plus_generator_new($1, $3); }
```

Comment:
[optional]

Factoring these out to `plus_generator` or something similar would make this a bit more tidy.